src/typeEvaluator/typeEvaluate.ts

```diff
@@ -62,14 +63,15 @@ const $warn = debug('typeEvaluator:evaluate::warn')
  * @beta
  */
 export function typeEvaluate(ast: ExprNode, schema: Schema): TypeNode {
+  $debug('evaluateQueryType.ast %O', ast)
   const parsed = walk({
     node: ast,
     scope: new Scope([], undefined, new Context(schema)),
   })
 
   $trace('evaluateQueryType.parsed %O', parsed)
   const optimized = optimizeUnions(parsed)
-  $trace('evaluateQueryType.optimized %O', optimized)
+  $debug('evaluateQueryType.optimized %O', optimized)
```

Comment:
Why debug instead of trace?

Reply:
So that you can get some minimal debug-information by turning on debug, you get the ast passed in and the resulting typenode